run-container/README.md

```diff
@@ -0,0 +1,133 @@
+# How to run the container for the Single cell RNA Seq Snap workflow (ScRNASeqSnap)
+
+We have generated Dockerfile and Definition file that contain all tools, packages, and dependencies necessary to run the code and analyses modules in the `sc-rna-seq-snap` repository. These are customized for `Rstudio/R v4.4.0` and `Seurat v4.4.0`.
+
+
+## To use the container in an R interactive session on HPC:
+
+From an interactive node on HPC, the user can open their session. Please modify memory and resources as needed for the analysis module to run.
+```
+bsub -P hpcf_interactive -J hpcf_interactive -n 2 -q standard -R "rusage[mem=16G]" -Is "bash"
+```
+
+## Load specific version of Singularity
+
+Please note that a version of Singularity is installed by default on all the cluster nodes at St Jude HPC. 
+Otherwise the user needs to ensure and load Singularity module by running:
+```
+module load singularity/4.1.1
+```
+
+
+1. Clone the repository
+```
+git clone https://github.com/stjude-dnb-binfcore/sc-rna-seq-snap.git
+```
+
+2. Pull the singularity container
```

Comment:
Mostly for my own clarity, is the rstudio_4.4.0_seurat_4.4.0_latest.sif expected to be loaded in a specific path within the repo? It seemed to me when I was running the tests that is was expected to be in the /sc-rna-seq-snap directory (not in any subdirectorires, like /sc-rna-seq-snap/run-container, for example).

Reply:
The container should be pulled while in the `root_dir`. I added a clarification here 574f668de7432d6099d68692145068f429c6267c.